---
float64 x
float64 y
float64 z
string task
bool rand